boards/arm64/intel_socfpga_agilex_socdk/intel_socfpga_agilex_socdk_defconfig

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# The Zephyr build from this defconfig is execpted to boot from
+# Intel Arm Trusted Firmware (ATF)
+# Boot Flow: BL21 -> BL31 -> Zephyr
+
+CONFIG_SOC_SERIES_AGILEX=y
+CONFIG_SOC_AGILEX=y
+CONFIG_BOARD_INTEL_SOCFPGA_AGILEX_SOCDK=y
+CONFIG_ARM_ARCH_TIMER=y
+
+# Zephyr Kernel Configuration
+CONFIG_XIP=n
+CONFIG_FLASH_SIZE=0
+CONFIG_FLASH_BASE_ADDRESS=0x0
+CONFIG_SRAM_BASE_ADDRESS=0x10000000
+CONFIG_SRAM_SIZE=256
+CONFIG_AARCH64_IMAGE_HEADER=y
```

Comment:
None of this should actually be needed.
- `CONFIG_XIP` is `n` by default
- `CONFIG_AARCH64_IMAGE_HEADER` is `y` by default now, 
- `CONFIG_SRAM_*` should be defaulted to whatever is in the DTS
- same for `CONFIG_FLASH_*`

Reply:
Noted. I will remove this when submit version 2 for review.